study2/index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Progate</title>
+    <link rel="stylesheet" href="stylesheet.css">
+  </head>
+  <body>
+    <!-- ここからHTMLを書き始めてください -->
+    <header>
+      <div class="header-container">
+        <div class="logo">
+          Progate
+        </div>
+        <div class="menu-list">
+          <ul>
+            <li>
+              <a href="#">プログラミングとは</a>
+            </li>
+            <li>
+              <a href="#">学べるレッスン</a>
+            </li>
+            <li>
+              <a href="#">お問い合わせ</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </header>
+    <main>
+      <div class="main-container">
+        <div class="message">
+          <h1>HELLO WORLD<span>.</span></h1>
+          <h2>プログラミングの世界へようこそ</h2>
+        </div>
+        <div class="lesson">
+          <h3>学べるレッスン</h3>
+          <div>
+            <img src="https://s3-ap-northeast-1.amazonaws.com/progate/shared/images/lesson/html/study/html.svg">
+            <p>HTML</p>
+          </div>
+          <div>
+            <img src="https://s3-ap-northeast-1.amazonaws.com/progate/shared/images/lesson/html/study/php.svg">
+            <p>PHP</p>
+          </div>
+          <div>
+            <img src="https://s3-ap-northeast-1.amazonaws.com/progate/shared/images/lesson/html/study/ruby.svg">
+            <p>Ruby</p>
+          </div>
+          <div>
+            <img src="https://s3-ap-northeast-1.amazonaws.com/progate/shared/images/lesson/html/study/swift.svg
+">
+            <p>Swift</p>
+          </div>
+        </div>
+        <div class="contact">
+          <h3>お問い合わせ</h3>
+          <p>メールアドレス（必須）</p>
```

Comment:
フォーム周り難しいですよね。ラベルタグがあるので活用していきましょう。
label の覆う方法と、input の id を label 側の for で参照する方法の２つの書き方があるので、是非調べてみてください。

```suggestion
　　　<label for="email">メールアドレス（必須）</label>
　　　<input type="email" id="email">
```